pySCG: updated to scope addressing Davids feedback, fixed formatting (#710)
ossf/wg-best-practices-os-developers: docs/Secure-Coding-Guide-for-Python/readme.md

# Secure Coding One Stop Shop for Python

An initiative by the OpenSSF to provide new Python programmers a resource to study secure coding in `CPython >= 3.9` with working code examples.

> Documentation is written in academic style to support security researchers while using in plain English to cater for an international audience.

Reviewer: ```suggestion
Documentation is written in academic style to support security researchers while using plain English to cater for an international audience.
```

> Python modules outside of the _Python Module Index_ [[Python 2023](https://docs.python.org/3.9/py-modindex.html)] are not specifically covered by this document.

Reviewer: I believe the order of words matters here to capture the intended meaning.

```suggestion
Python modules outside of the _Python Module Index_ [[Python 2023](https://docs.python.org/3.9/py-modindex.html)] are specifically not covered by this document.
```

> Python modules outside of the _Python Module Index_ [[Python 2023](https://docs.python.org/3.9/py-modindex.html)], languages such as `HTML` and `JavaScript` are avoided.

Reviewer 2: ```suggestion
Python modules outside of the _Python Module Index_ [[Python 2023](https://docs.python.org/3.9/py-modindex.html)] are not specifically covered by this document. We do sometimes note where outside modules are necessary to address a type of vulnerability, and encourage users of those modules to consult those modules' documentation.
```

Reviewer 2: The note about HTML & JavaScript isn't really the issue. The point is that this document only discusses the built-in modules. Presumably JSON *would* be in scope, because there's a built-in module for JSON.

Author: @david-a-wheeler , the background story to this learning resource is that we have designers spending 40 hours going through a ton of text to learn secure coding in languages such as JAVA. Any chance we can shorten that sentence?

Reviewer 2: ```suggestion
Python modules outside of the _Python Module Index_ [[Python 2023](https://docs.python.org/3.9/py-modindex.html)] are not specifically covered by this document.
```

Reviewer 2: How about this? The issue isn't really HTML (there's an `html` module in Python's built-ins) - it's that this particular document only focuses on the built-ins.

Author: Much better thank you.

Since I am not native English speaking I struggle to understand the difference between "avoiding" and "not specifically covered". We try our best to only list "inclusions" in order to have ever expanding lists of "exclusions". Not sure how to best phrase that in this introduction.

We are missing  a contribution document,  I will get onto creating that. In it I would outline the following in regards to explaining why we avoid modules outside of the standard lib:
* Code examples may end up as module recommendations "just use..."
* Actual issue is not explained in code.
* Don't want to end up writing module documentation, bug-fixes or workarounds.
* Life Cycle related issues
* Trust related issues

We also want to keep the main pySCG readme.md look "inviting" with a minimum of content and links in its introduction section.
